pkg/cloud/metadata_k8s.go

```diff
@@ -32,6 +33,21 @@ var DefaultKubernetesAPIClient = func() (kubernetes.Interface, error) {
 	return clientset, nil
 }
 
+// Get testing kubernetes APIclient
+var TestingKubernetesAPIClient = func() (kubernetes.Interface, error) {
+	// Get configuration from config file
+	config, err := clientcmd.BuildConfigFromFlags("", "/root/.kube/config")
+	if err != nil {
+		fmt.Println("ERROR building configuration:", err)
```

Comment:
why are we just printing? why are we not returning?

Reply:
fixed